### ISSUE TITLE
Sending MIDI notes to Yosemite not working

### ISSUE THREAD

**Issue by ovicin** [jardous/bluefruit_midi_proto#1]
I tried your sketch on the Bluefruit SPI version and I was not able to send MIDI notes to Yosemite or IOS, it worked though with an Android BLE MIDI application. You tried your sketch with Apple products? Is it working well on your side?
Thanks

**Comment by jardous** [comment]
yeah... that is the thing... I can see the MIDI device available, can connect to it, but how to send the data is not working there. Need help with that too. Please feel free contribute.

**Comment by ovicin** [comment]
I had a second look on the bluefruit documentation but is not clear for me if using the **BLUEFRUIT_MODE_DATA** will write the data to the MIDI characterisc or should be used the command **AT+GATTCHAR**

```
AT+GATTCHAR
Gets or sets the value of the specified custom GATT characteristic (based on the index ID returned when the characteristic was added to the system via AT+GATTADDCHAR).

Codebase Revision: 0.3.0

Parameters: This function takes one or two comma-separated functions (one parameter = read, two parameters = write).

    The first parameter is the characteristic index value, as returned from the AT+GATTADDCHAR function. This parameter is always required, and if no second parameter is entered the current value of this characteristic will be returned.
    The second (optional) parameter is the new value to assign to this characteristic (within the MIN_SIZE and MAX_SIZE limits defined when creating it).

Response: If the command is used in read mode (only the characteristic index is provided as a value), the response will display the current value of the characteristics.  If the command is used in write mode (two comma-separated values are provided), the characteristics will be updated to use the provided value.
```

I think that using **BLUEFRUIT_MODE_DATA** will write the data to the UART characteristic.

**Comment by ovicin** [comment]
It works for me know by setting the value to the characteristic with AT+GATTCHAR, I receive the notes on an IOS device.
Note on: AT+GATTCHAR=1,AF-80-90-3F-64
Note off: AT+GATTCHAR=1,AF-80-80-0-0

**Comment by jardous** [comment]
yes! That is the break through! I can confirm that it works for me (just you did not specify the note to off - should be AT+GATTCHAR=1,AF-80-80-3F-00)

Thanks a lot!

**Comment by jorgertrias** [comment]
Hi, I'm building a midi foot controller with Arduino. I am interested in offering BLE MIDI connection, which recognizes an IOS device.
I tried connecting a module HM 10 (CC2540 chip) Bluetooth but the iPad does not recognize it as MIDI device BLE.
With Adafruit Friend Bluefruit LE module UART can I get it?
I currently use the MIDI Arduino library (https://github.com/FortySevenEffects/arduino_midi_library/)
Thanks in advance from Spain!!
